contracts/Registry.vy

```diff
@@ -148,18 +166,29 @@ def _get_rates(_pool: address) -> uint256[MAX_COINS]:
     rates: uint256[MAX_COINS] = empty(uint256[MAX_COINS])
     base_pool: address = self.pool_data[_pool].base_pool
     if base_pool == ZERO_ADDRESS:
-        rate_method_id: Bytes[4] = slice(self.pool_data[_pool].rate_method_id, 28, 4)
+        rate_method_id: bytes32 = self.pool_data[_pool].rate_method_id
```

Comment:
What is the rationale for changing this to `bytes32` and slicing at line 190? That will mean the `slice` happens as many times as we iterate and the function becomes more expensive.  Or is there something I'm not seeing?

Reply:
Absolutely right! Noob move, fixed in 3985915